packages/ui/src/helpers/accountSettings/utils.ts

```diff
@@ -1,37 +1,35 @@
-import * as Auth from '@aws-amplify/auth';
+import {
+  updatePassword,
+  deleteUser as deleteCognitoUser,
+} from '@aws-amplify/auth';
```

Comment:
Don't want to import the whole module here, but will think of a better naming convention